lib/fastlane/plugin/wpmreleasetoolkit/actions/common/close_milestone_action.rb

```diff
@@ -43,7 +43,7 @@ def self.available_options
                                    optional: false,
                                        type: String),
           FastlaneCore::ConfigItem.new(key: :milestone,
-                                        env_name: "GHHELPER_MILESTORE",
+                                        env_name: "GHHELPER_MILESTONE",
```

Comment:
Careful accepting this fix as it would need anyone who is depending on this ENV var to update their development environment accordingly

Reply:
@loremattei – I think this one is set in every usage anyway during the `finalize_release` lanes, but WDYT?

Reply:
yep! It is. I think we're ok with merging this.